.github/workflows/firebase_deploy_main.yml

```diff
@@ -37,6 +37,34 @@ jobs:
           args: deploy -P test --only hosting:project-dashboard-test
         env:
           FIREBASE_TOKEN: ${{ secrets.FIREBASE_TOKEN }}
+      # Only run the coverage once
+      - name: get value percentage and print
+        run: |
+          # var REF = 'refs/pull/27/merge.json';
+          REF=${{ github.ref }}
+          # console.log('github.ref: ' + REF);
+          echo "github.ref: $REF"
+          # var PATHS = REF.split('/');
+          IFS='/' read -ra PATHS <<< "$REF"
+          # var COVERAGEBADGE = PATHS[1] + PATHS[2];
+          COVERAGEBADGE="${PATHS[1]}_${PATHS[2]}"
+          echo $COVERAGEBADGE
+          echo "COVERAGEBADGE=$(echo ${COVERAGEBADGE})" >> $GITHUB_ENV
```

Comment:
This par (line 43-52) is not needed for the main branch as the json filename is hard coded at line 63